CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.2](https://github.com/brave/brave-ios/releases/tag/v1.57.2)
+
+- Improved performance of closing tabs for sites where Window.open is called many times. ([#8060](https://github.com/brave/brave-ios/issues/8060))
+- Fixed tab selection UI when switching between private and regular modes. ([#8146](https://github.com/brave/brave-ios/issues/8146))
```

Comment:
Maybe mention what the issue was? Sounds like there was issues with the wrong UI being shown and not being able to switch back to regular mode from private mode in certain cases.